### PR TITLE
Added poutils section to README

### PR DESCRIPTION
**Pull request by Seluj78**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ BSD 3-clause
 
 Pull request are welcome.
 
+### Padpo is part of poutils!
+
+[Poutils](https://pypi.org/project/poutils) (`.po` utils) is is a metapackage to easily install usefull Python tools to use with po files
```
Comment (vpoulailleau) [inline README.md:28]:
```suggestion
[Poutils](https://pypi.org/project/poutils) (`.po` utils) is a metapackage to easily install useful Python tools to use with po files
```

File: README.md [hunk 2]
```diff
@@ -23,6 +23,12 @@ BSD 3-clause
...
+and `padpo` is a part of it! Go check out [Poutils](https://pypi.org/project/poutils) to discover the other useful tools for `po` file related translation!
```
Comment (vpoulailleau) [inline README.md:29]: To remove redundancy

```suggestion
and `padpo` is a part of it! Go check out [Poutils](https://pypi.org/project/poutils) to discover the other tools!
```

File: README.md [hunk 3]
```diff
@@ -23,6 +23,12 @@ BSD 3-clause
 
 Pull request are welcome.
 
+### Padpo is part of poutils!
```
Comment (vpoulailleau) [inline README.md:26]:
```suggestion
## Padpo is part of poutils!
```

**Review by vpoulailleau — CHANGES_REQUESTED** [review]
Thanks! Some small modifications though. I'll answer to the issue in `poutils` later, but it seems a good idea.

**Comment by Seluj78** [comment]
Thanks @vpoulailleau !

**Review by vpoulailleau — APPROVED** [review]

**Pull request merged**